locale.c

```diff
@@ -2485,9 +2485,9 @@ S_find_locale_from_environment(pTHX_ const unsigned int index)
 
         /* If no corresponding environment variable, see if LANG exists.  If
          * so, use it. */
-        const char * default_name = PerlEnv_getenv("LANG");
-        if (default_name && strNE(default_name, "")) {
-            return default_name;
+        const char * LANG = PerlEnv_getenv("LANG");
+        if (LANG && strNE(LANG, "")) {
+            return LANG;
```

Comment:
I don't think the variable name should be all upper-case, it screams "macro" to me.

Maybe `env_lang` or `env_LANG` (I prefer the first).

Reply:
changed to env_lang